index.html

```diff
@@ -1,27 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="data.js"></script>
-  <script src="index.js"></script>
   <style>
     img {
       height: 300px;
     }
   </style>
   <title>JS Mini Challenge: DOM Manipulation</title>
 </head>
+
 <body>
   <h1 id="header">Welcome to Ian's Liverpool FC Fan Club</h1>
   <div class="player">
-    <img src="https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg" alt="{player name}">
+    <img src="https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg"
+      alt="{player name}">
     <h2>{Player Name}</h2>
     <em>{Player Nickname}</em>
     <h4>Great Goals</h4>
     <ul id="goals">
       <!-- add links to goals here -->
     </ul>
   </div>
+  <script src="data.js"></script>
```

Comment:
For the `data.js` file you don't actually need move this file to the body, since none of the code in that file references any DOM elements. It won't break, and if you want to organize all the script tags at the bottom of the `<body>` that's totally fair - just wanted to point that out!